pkg/api/mlflow/dao/models/log.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+// Log represents a row of the `logs` table.
+type Log struct {
+	ID        uint   `gorm:"primaryKey;autoIncrement"`
+	Value     string `gorm:"not null"`
+	RunID     string `gorm:"column:run_uuid;not null;index"`
```

Comment:
doesn't really matter here, but the gorm annotation for index needs to have the shared name `idx_run_timestamp`

Reply:
previously there was index on runID + timestamp as a compound index (which requires both fields to reference the same index name). Now it is just separate index on Timestamp and RunID -- but the name for the Timestamp one looks like a compound.